packages/nuxt/src/runtime/plugins/sentry.server.ts

```diff
@@ -14,9 +14,17 @@ export default defineNitroPlugin(nitroApp => {
       }
     }
 
+    const currentScope = Sentry.getCurrentScope();
```

Comment:
l: Is there a particular reason you're creating a binding for the current scope here instead of just calling `getCurrentScope().setTransactionName()` directly?